packages/repository/src/query.ts

```diff
@@ -106,7 +109,7 @@ export interface Filter {
   /**
    * Sorting order for matched entities
    */
-  order?: Order[];
+  order?: string[];
```

Comment:
I am worried that this change will allow people to accidentally use invalid values for `order` and the TypeScript compiler will not catch that problem. Can we preserve this type? Unless I am missing something?

Reply:
There are a few issues around this signature:

1. Order is defined as `{[field: string]: Direction}`. That means we can have `{a: 'ASC', b: 'DESC'}. Do we translate it to `ORDER BY A ASC, B DESC`? If so, can we avoid `Order[]`?

2. `string[]` is not very intuitive. I use it internally for now to avoid translation to legacy juggler `order` format.

3. The `FilterBuilder` takes `Order` for type safety.

Reply:
I see, I think I got confused about the difference between `Order` and `Direction`, as it seems to be the same thing. Maybe `Ordering` would be a better type name?

In the light of your comment, I am ok to keep the current implementation, but please add a comment explaining why we are using `string[]` type here and perhaps add a link pointing to our docs where people can learn more about valid values.